myuw/static/js/card/calendars/academic_calendar.js

```diff
@@ -82,8 +85,8 @@ var AcademicCalendar = {
     },
```

Comment:
@fanglinfang My changes in this file are all in the `filter_all` and `filter_breaks` functions.